Entities/TDExplosive.cpp

```diff
@@ -38,7 +38,10 @@ namespace RTE {
 	int TDExplosive::ReadProperty(const std::string_view &propName, Reader &reader) {
 		// TODO: Consider removing DetonationSound as GibSound already exists and could be used in its place
 		if (propName == "DetonationSound") {
-			reader >> m_GibSound;
+			if (!m_GibSound) {
```

Comment:
This should overwrite GibSound if it already exists. i.e.
```cpp
if (!m_GibSound) { m_GibSound = new SoundContainer; }
reader >> m_GibSound;
```

Reply:
Just to confirm, this should be the same for `GibSound` in MOSRotating.cpp, yes? i.e. If either is defined after the other, it should overwrite the existing `GibSound` when redefined?

Reply:
Uhh yeah, I'd say so. If I'm not mistaken, the old code for both of them was `reader >> m_GibSound`, right? That means that whatever is read in last is the final one used, I'd say stick with that so there's no undesired functionality change here.